app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Hashable
+
+
 class Dictionary:
-    pass
+    DEFAULT_CAPACITY = 8
+    LOAD_FACTOR = 2 / 3
```

Comment:
The LOAD_FACTOR is defined using integer division, which will result in 0. It should be a float, so use 2/3.0 or 2.0/3 to ensure it is a float.